db/seeds.rb

```diff
@@ -1,7 +1,33 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
-#   Character.create(name: "Luke", movie: movies.first)
+require 'faker'
+
+puts "Cleaning database..."
+User.destroy_all
+Property.destroy_all
+
+puts "🌱 Seeding data..."
+puts "👩🏽‍🦱 Creating users with faker..."
+
+10.times do
+  fn = Faker::Name.first_name
```

Comment:
Please, use meaningful names (`first_name` rather than `fn`).
Same thing for `ln` (should be `last_name`).